### PR TITLE
vendor: update google.golang.org/grpc to 1.0.5

### PR DESCRIPTION
**Pull request by mvdan**
We were using a slightly older version. These do have stable versions,
so pin the latest.

### CODE REVIEW

**Comment by buger** [comment]
What was the previous version in terms of semver?

**Comment by mvdan** [comment]
None; we were using HEAD as of mid november, I assume that's when @matiasinsaurralde's coprocess stuff went in.

The bump in time is of around 15 days.

**Comment by buger** [comment]
Yes, but what we have commit id, so it is possible to get version which was vendored. I wanted to check the changelog.

**Comment by mvdan** [comment]
Sure, you can do that. I'm not sure what your question is.

**Comment by buger** [comment]
Ok, let me rephrase it. How do you ensure that updating this package is safe?

**Comment by mvdan** [comment]
I can't - this is why I'm doing this PR now and not a week before the 2.4 release.

In any case, using a stable version is better than using a random HEAD.

**Comment by buger** [comment]
Here is changelog https://github.com/grpc/grpc-go/compare/b13ef794997191ec801db90a42c50034df91850a...708a7f9f3283aa2d4f6132d287d78683babe55c8

**Comment by buger** [comment]
Seems like mostly fixes and documentation updates. Our version is between 1.0.4 and 1.0.5.
